src/Facets/WhitelistManagerFacet.sol

```diff
@@ -4,92 +4,78 @@ pragma solidity ^0.8.17;
 import { LibDiamond } from "../Libraries/LibDiamond.sol";
 import { LibAccess } from "../Libraries/LibAccess.sol";
 import { LibAllowList } from "../Libraries/LibAllowList.sol";
+import { IWhitelistManagerFacet } from "../Interfaces/IWhitelistManagerFacet.sol";
 import { CannotAuthoriseSelf } from "../Errors/GenericErrors.sol";
 
-/// @title Dex Manager Facet
+/// @title Whitelist Manager Facet
 /// @author LI.FI (https://li.fi)
-/// @notice Facet contract for managing approved DEXs to be used in swaps.
-/// @custom:version 1.0.2
-contract DexManagerFacet {
-    /// Events ///
-
-    event DexAdded(address indexed dexAddress);
-    event DexRemoved(address indexed dexAddress);
-    event FunctionSignatureApprovalChanged(
-        bytes4 indexed functionSignature,
-        bool indexed approved
-    );
-
+/// @notice Facet contract for managing whitelisted addresses for various protocol interactions.
+/// @custom:version 1.0.3
```

Comment:
Feels a bit funny that we rename the contract and then continue the versioning.  Maybe only to me?

How about we start at 1.0.0 (new contract) or add a comment that it was renamed from x (V.1.0.2) to y (V1.0.3) ? 
I am also OK if we just do it this way. Just wanted to raise my concern.

Reply:
I was also thinking about this since we’re rebranding, I initially leaned toward continuing the versioning. But you’re right it’s technically a new contract so starting from 1.0.0 makes more sense. Great catch 👍🏻